src/main/java/gov/cms/madie/terminology/service/FhirTerminologyService.java

```diff
@@ -122,7 +124,49 @@ private List<QdmValueSet.Concept> getValueSetConcepts(
   }
 
   public List<CodeSystem> getAllCodeSystems() {
-    return codeSystemRepository.findAll();
+    // remove items that are marked as not present in vsac to cut expense
+    List<CodeSystemEntry> codeSystemEntries = mappingService.getCodeSystemEntries().stream().filter(codeSystemEntry -> !codeSystemEntry.getOid().contains("NOT.IN.VSAC")).toList();
+    List<CodeSystem> codeSystems = codeSystemRepository.findAll();
+    List<CodeSystem> transformedResults = new ArrayList<>(Collections.emptyList());
```

Comment:
Passing in the empty list to the ArrayList constructor doesn't seem necessary. 

From the [ArrayList javadoc](https://docs.oracle.com/en%2Fjava%2Fjavase%2F21%2Fdocs%2Fapi%2F%2F/java.base/java/util/ArrayList.html) on passing in a Collections to the constructor:
> Constructs a list containing the elements of the specified collection, in the order they are returned by the collection's iterator.

Thus we are making an empty List from emptyList.